shared/js/ui/models/whitelist.es6.js

```diff
@@ -44,9 +50,29 @@ Whitelist.prototype = window.$.extend({},
           value: true
         }
         })
+
+        this.setWhitelistFromSettings()
       }
 
       return domain
+    },
+
+    setWhitelistFromSettings: function () {
+      let self = this
+      this.fetch({getSetting: {name: 'whitelisted'}}).then((whitelist) => {
+        whitelist = whitelist || {}
+        let wlist = Object.keys(whitelist)
+        wlist.sort()
+        
+        // Publish whitelist change notification via the store
+        // used to know when to rerender the view
+        self.set('list', wlist)
+        
+        // set() sealed both the model's list and wlist
+        // unpack and wrap to allow adding/removing elements
+        // in the model property
+        self.list = [...wlist]
```

Comment:
It's not pretty but I need to redo the assignment here because, as explained in the comment, set() seals both model.list and the value, wlist.
So even if I pass [...wlist] to unpack it inside set (like this: self.set('list', [...wlist]) it's useless because it gets sealed again.